chat/uikit/ios.md

```diff
@@ -0,0 +1,386 @@
+---
+title: Skygear Chat iOS UIKit
+---
+
+[[toc]]
+
+## About UIKit
+UIKit provides the `SKYChatConversationViewController`, the main UI of a conversation.
+The view of `SKYChatConversationViewController` is `SKYChatConversationView`. The view consists of the following components.
+
+1. Title Bar
+2. Avatar (Outgoing)
+3. Message (Outgoing)
+4. Avatar (Incoming)
+5. Message (Incoming)
+6. Camera Button
+7. Text Input
+8. Voice Button
+
+![ConversationView Overview](/assets/chat/uikit/ios/conversation-view-overview.png)
+
+
+## Prerequisite
+### Step 1: Add Skygear and Skygear Chat to your project
+
+Please follow the steps at [Quick Start](https://docs.skygear.io/guides/chat/quick-start/ios/).
+
+### Step 2: Add Skygear Chat UIKit to your project
+
+
+1. add `pod 'SKYKitChat/UI', '~> 1.1'` in `Podfile`.
+
+```bash
+pod 'SKYKitChat/UI', '~> 1.1'
```

Comment:
Just afraid we will forget to update this version, we should be a newer version that include the ui kit update. Should be 1.2 or 1.4? We can discuss.

Reply:
@rickmak  any thought on versioning?